packages/strapi-plugin-users-permissions/documentation/1.0.0/overrides/users-permissions-User.json

```diff
@@ -0,0 +1,281 @@
+{
+  "paths": {
+    "/auth/local/register": {
+      "post": {
```

Comment:
This route should have an override on security as it doesn't require authentication to use.

Example to remove authentication within swagger:

```
  "paths": {
    "/auth/local/register": {
      "post": {
        "tags": ["Authentication"],
        "security": [],
        "externalDocs": {
          "description": "Find out more in the strapi's documentation",
```

Reply:
![image](https://user-images.githubusercontent.com/8593673/49746811-45c15600-fca2-11e8-9717-e81654fab9c1.png)

Reply:
Fixed with merged PR https://github.com/strapi/strapi/pull/2462